.gitignore

```diff
@@ -6,10 +6,10 @@ lib-cov
 *.out
 *.pid
 *.gz
-bin
 pids
 logs
 results
+bin/index.js
```

Comment:
I'd say to keep ignoring the bin folder, not just index.js

Reply:
The bin folder contains a base TS type. which is why its there.